exporter/signalfxexporter/README.md

```diff
@@ -74,6 +74,9 @@ The following configuration options can also be configured:
   processor is enabled in the pipeline with one of the cloud provider detectors
   or environment variable detector setting a unique value to `host.name` attribute
   within your k8s cluster. And keep `override=true` in resourcedetection config.
+- `allowed_dimension_chars`: (default = `"_-"`) A string of characters 
+that are allowed to be used as a dimension key in addition to alphanumeric 
+characters. If a character is not allowed, it is replace with a `_`.
```

Comment:
nits:

`in a dimension key in addition to...`

```suggestion
characters.  Each nonalphanumeric dimension key character that isn't in this string will be replaced with a `_`.
```

or at least s/replace/replaced